src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -65,6 +68,44 @@ public void RegisterActor<TActor>(Func<ActorTypeInformation, ActorService> actor
             this.actorManagers[actorTypeInfo.ActorTypeName] = new ActorManager(actorService);
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public void UseActorSettings(ActorSettings settings)
```

Comment:
Thinking more about it, I am more inclined towards ConfigureSettings method as it allows us(in future) to use ActorSettings with non null defaults and users can modify whatever they want on the object.